cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -176,4 +194,8 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         assessmentApplication.perform_assessment("low", [stakeHolder]);
         assessmentApplication.verifyStatus("assessment", "Completed");
     });
+
+    it("Assess archetype", function () {
+        archetype.perform_assessment("low", [stakeHolder], [stakeHolderGroup]);
```

Comment:
Add the following statement to verify archetype assessment is complete.
archetype.validateAssessmentField("Low");

Reply:
Done